### PR TITLE
IsDeleted returns true if DeletedAt is not zero

### LINKED ISSUE #84: [Paranoid] IsDeleted should also check if time.Time is zero

### PR DESCRIPTION
**Pull request by EtienneM**
Fix #84

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## v5.5.8 (Apr 16 2019)
+
+* [paranoid] IsDeleted returns true if DeletedAt is zero
```
Comment (Soulou) [inline CHANGELOG.md]: is NOT zero

**Comment by Soulou** [comment]
is NOT zero (PR name is wrong)

File: CHANGELOG.md (hunk 1 above)
Comment (EtienneM) [reply CHANGELOG.md]: Woupsy

**Comment by Soulou** [comment]
LGTM

**Pull request merged**